### ISSUE TITLE
Incorrect UINT32 definition

### ISSUE THREAD

**Issue by falkTX** [baconpaul/BaconPlugs#51]
Reporting here as you might be quick with the fix, let me know if you rather have a PR.

Basically this:

```
BaconPlugs/libs/open303-code/Source/DSPCode/GlobalDefinitions.h:51:23: error: conflicting declaration ‘typedef long unsigned int UINT32’
   51 | typedef unsigned long UINT32;
      |                       ^~~~~~
...

/usr/share/mingw-w64/include/basetsd.h:35:24: note: previous declaration as ‘typedef unsigned int UINT32’
   35 |   typedef unsigned int UINT32,*PUINT32;
      |                        ^~~~~~
```

Really not sure why an `unsigned long` would be treated as `uint32` type, doesn't seem correct to me.
Any reason for this? It conflicts with the mingw/Windows UINT32 type.

**Comment by falkTX** [comment]
IIRC on windows 32bit systems `sizeof(long) == sizeof(int)` so that could explain things, the code was made back when 32bit was more commonplace and just never updated.

**Comment by baconpaul** [comment]
Oh yeah that stuff is kinda not finished but I’m
Happy for prs! Trying to get surge xt into shape next week so bit engaged with that

**Comment by falkTX** [comment]
Closing as this was fixed in c7eb417fa18fd87fe4e49eee22af1ed51547a4ce
